etc/helm/pachyderm/templates/enterprise-server/postgres-secret.yaml

```diff
@@ -0,0 +1,16 @@
+{{- /*
+SPDX-FileCopyrightText: Pachyderm, Inc. <info@pachyderm.com>
+SPDX-License-Identifier: Apache-2.0
+*/ -}}
+{{- if and .Values.enterpriseServer.enabled (not .Values.postgresql.enabled) -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    app: postgresql
+    suite: pachyderm
+  name: postgres # Must match secretname specified for postgres password in pachd deployment, enterprise and pg bouncer
```

Comment:
Doesn't this conflict with `pachd/postgres-secret.yaml`?

Reply:
They'll only ever be deployed separately